### DISCUSSION TITLE
Size control in different media queries

### DISCUSSION THREAD

**Discussion by SergeyBiryukov34** [mantinedev/mantine#322] [Q&A] [answered]
Good day! Please tell me how to resize components at different media queries?
![Screenshot_2](https://user-images.githubusercontent.com/59689428/138076624-8484fc21-0948-47b8-a9be-19e2abc964fd.png)

**Comment by rtivital** [comment, marked as answer]
We currently do not support that, you can add styles with media queries via Styles API and createStyles function
